sdk/typespec/typespec_client_core/src/http/pipeline.rs

```diff
@@ -30,13 +32,17 @@ use std::sync::Arc;
 /// cannot be enforced by code). All policies except Transport policy can assume there is another following policy (so
 /// `self.pipeline[0]` is always valid).
 #[derive(Debug, Clone)]
-pub struct Pipeline {
+pub struct Pipeline<F: Format> {
```

Comment:
After our design discussion, I believe we agreed we'd keep `Pipeline` returning a `ResponseBody` or something, and then each client method would have to deserialize that (async) into a `Response<T, F>`, right?